judge/views/two_factor.py

```diff
@@ -67,6 +69,7 @@ def post(self, request, *args, **kwargs):
     def get_context_data(self, **kwargs):
         context = super(TOTPEnableView, self).get_context_data(**kwargs)
         context['totp_key'] = self.profile.totp_key
+        context['scratch_codes'] = '\n'.join(json.loads(self.profile.scratch_codes))
```

Comment:
I think for consistency with the Ajax view, the `'\n'.join` should be moved into the template.

See: [Jinja join filter](https://jinja.palletsprojects.com/en/2.11.x/templates/#join).